templates/prelogin/base.html

```diff
@@ -205,6 +205,32 @@ <h3 class="modal-title">
                 </div>
             </div>
         </div>
+        <div id="contactUsFormErrors"
+             class="modal fade">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">{% trans 'Close' %}</span></button>
+                        <h3 class="modal-title">
+                            {% trans "Ooops!" %}
+                        </h3>
+                    </div>
+                    <div class="modal-body">
+                        <p>
+                            {% comment %}
+                                There is only one field that can result in errors,
+                                so it's safe to assume the problem is the e-mail
+                                address.
```

Comment:
This is currently true, and will work for now, but it's pretty far away from the form definition where other validation conditions may be added in the future. Would it be very hard to pass the errors in here rather than assuming that this is the only possible error?

In summary, this feel fragile and could easily trip someone up in the future. I'm also thinking about the person who *did* enter an email address and is still getting this error for some inexplicable reason.